auth-server/src/resolvers/mutation/changeName.ts

```diff
@@ -8,9 +8,9 @@ interface argsType {
 }
 
 export default async (parent, { newName }: argsType, ctx: Context) => {
-	const token = getTokenFromReq(ctx.req)
+	let token = getTokenFromReq(ctx.req)
```

Comment:
We need the same restrictions as for signup, not allow <6 char.

Reply:
done